src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -246,11 +244,14 @@ const _sendUserOp = async (
       waitForDeployment: false,
     })) as UserOperation; // TODO support entrypoint v0.7 accounts
   } catch (error) {
-    return {
+    const errorMessage = wrapError(error, "Bundler").message;
+    const erroredTransaction: ErroredTransaction = {
       ...queuedTransaction,
       status: "errored",
-      errorMessage: wrapError(error, "Bundler").message,
-    } satisfies ErroredTransaction;
+      errorMessage,
+    };
+    job.log(`Failed to populate transaction: ${errorMessage}`);
+    return erroredTransaction;
```

Comment:
Unrelated fix, I previously removed the job.log() here but now I want it back. It's important for us to know if an error was thrown during simulation/estimation or when sending the tx.